IDEHelper/Compiler/BfIRBuilder.cpp

```diff
@@ -1572,12 +1568,8 @@ void BfIRBuilder::SetBackend(bool isBeefBackend)
 
 void BfIRBuilder::RemoveIRCodeGen()
 {
-	if (mIRCodeGen != NULL)
-	{
-		mIRCodeGen->mStream = NULL;
```

Comment:
This setting to NULL seemed unnecessary, as this function is cleaning up the `mIRCodeGen` pointer entirely and this is a member variable of the class.